sslr-flex-toolkit/pom.xml

```diff
@@ -46,35 +49,18 @@
           </archive>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>jarjar-maven-plugin</artifactId>
```

Comment:
do we not need this plugin?

Reply:
oh, we use shade instead of jar jar binx

Reply:
yes